operations/iuf/IUF.md

```diff
@@ -0,0 +1,509 @@
+  -bm BOOTPREP_CONFIG_MANAGEMENT [BOOTPREP_CONFIG_MANAGEMENT ...], --bootprep-config-management BOOTPREP_CONFIG_MANAGEMENT [BOOTPREP_CONFIG_MANAGEMENT ...]
+                        List of `sat bootprep` config files for management NCN nodes.
+  -bpcd BOOTPREP_CONFIG_DIR, --bootprep-config-dir BOOTPREP_CONFIG_DIR
+                        directory containing bootprep configuration files.  The expected layout would be:
+                                $(BOOTPREP_CONFIG_DIR)/product_vars.yaml
+                                $(BOOTPREP_CONFIG_DIR)/bootprep/compute-and-uan-bootprep.yaml
+                                $(BOOTPREP_CONFIG_DIR)/bootprep/management-bootprep.yaml
+  -rv RECIPE_VARS, --recipe-vars RECIPE_VARS
+                        location of the recipe_vars.yaml file (aka product_vars.yaml), used by `sat bootprep`
+  -sv SITE_VARS, --site-vars SITE_VARS
+                        Path to site parameters file.  Default is ${RSM_BASE_DIR}/$IUF_ACTIVITY_SESSION}/site_vars.yaml
+  -um UPDATE_METHOD_MANAGEMENT, --update-method-management UPDATE_METHOD_MANAGEMENT
+                        Method to update the management nodes.  (rolling) 'reboot' or (rolling) 'in_place'
+                        (update running nodes).  Defaults to 'reboot'
+  -uc UPDATE_METHOD_MANAGED, --update-method-managed UPDATE_METHOD_MANAGED
+                        Method to update the managed nodes.  (rolling) 'reboot' or (rolling) 'in_place'
+                        (update running nodes).  Defaults to 'reboot'
+  --limit-managed-nodes LIMIT_MANAGED_NODES [LIMIT_MANAGED_NODES ...]
+                        Override list used in rolling stage to assist with repairing broken managed
+                        nodes.
+  --limit-management-nodes LIMIT_MANAGEMENT_NODES [LIMIT_MANAGEMENT_NODES ...]
+                        Override list used in rolling stage to assist with repairing broken management
+                        nodes.
+  -mrp MASK_RECIPE_PRODS [MASK_RECIPE_PRODS ...], --mask-recipe-prods MASK_RECIPE_PRODS [MASK_RECIPE_PRODS ...]
+                        If product_vars was specified, mask the versions found within product_vars for
+                        the products specified, So versions for the specified products would be pulled
+                        from the product catalog.  Note that the versions found in the `--site-vars` (or
+                        the version being installed) would override it anyway.
+```
+
+These [examples](examples/iuf_run.md) highlight common use cases of `iuf run`.
+
+#### `abort`
+
+The `abort` subcommand is specified by the administrator to end a paused IUF session instead of attempting to resume or restart it.
+
+The following arguments may be specified when invoking `iuf abort`:
+
+```bash
+usage: << TODO >>
+```
+
+These [examples](examples/iuf_abort.md) highlight common use cases of `iuf abort`.
+
+#### `resume`
+
+The `resume` subcommand is specified by the administrator to resume a paused IUF session.
```

Comment:
This and resume may not be functional before the release.

Reply:
Put a comment in noting that implementation is pending.